sdk/identity/Azure.Identity/src/AzureIdentityEventSource.cs

```diff
@@ -31,6 +31,10 @@ internal sealed class AzureIdentityEventSource : AzureEventSource
         private const int ProcessRunnerErrorEvent = 14;
         private const int ProcessRunnerInfoEvent = 15;
         private const int UsernamePasswordCredentialAcquireTokenSilentFailedEvent = 16;
+        private const int TenantIdDiscoveredAndNotUsedEvent = 17;
+        private const int TenantIdDiscoveredAndUsedEvent = 18;
+        internal const string TenantIdDiscoveredAndNotUsedEventMessage = "The service provided an alternate tenantId from what was configured on the credential, but the configured value was used authorization. Configured TenantId: {0}, Alternate TenantId {1}";
```

Comment:
```suggestion
        internal const string TenantIdDiscoveredAndNotUsedEventMessage = "A token was request for a different tenant than was configured on the credential, but the configured value was used since multi tenant authentication has been disabled. Configured TenantId: {0}, Requested TenantId {1}";
```